test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -357,7 +395,7 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
 	}
 
 	if (smart == false)
```

Comment:
it should not. if smart opcode is not defined in switch then smart = false again